src/funding/paylater/config.jsx

```diff
@@ -27,10 +27,18 @@ export function getPaylaterConfig() : FundingSourceConfig {
 
             let label = <Text>Pay Later</Text>;
 
+            if (paylater?.products?.paylater?.eligible && paylater?.products?.paylater?.variant === 'DE') {
+                label = <Text>Später Bezahlen</Text>;
+            }
+
             if (paylater?.products?.payIn4?.eligible) {
```

Comment:
I would put this above the previous one.  Use it as the default and override based on country.